test/e2e/persistentips_test.go

```diff
@@ -312,3 +317,7 @@ func removeFinalizersPatch() ([]byte, error) {
 	}
 	return json.Marshal(patch)
 }
+
+func getIPsFromVMIStatus(vmi *kubevirtv1.VirtualMachineInstance) ([]string, error) {
```

Comment:
this is just for secondaries since it passes the secondary network interface name let's call this `secondaryNetworkVMIStatusIPs`

so we have 
```golang
ipsFrom: secondaryNetworkVMIStatusIPs
```

Reply:
done